create_test_cluster.py

```diff
@@ -11,7 +11,7 @@
 import time
 from argparse import Namespace
 
-VALID_OPERATORS = ["druid", "hbase", "hive", "kafka", "nifi", "opa", "spark", "superset", "trino", "zookeeper"]
+VALID_OPERATORS = ["druid", "hbase", "hive", "kafka", "nifi", "opa", "spark", "superset", "trino", "zookeeper", "airflow"]
```

Comment:
Please keep the alphabetical order:
```suggestion
VALID_OPERATORS = ["airflow", "druid", "hbase", "hive", "kafka", "nifi", "opa", "spark", "superset", "trino", "zookeeper"]
```